src/components/GalleryPage/Winners.js

```diff
@@ -74,35 +75,41 @@ function Winners() {
 	}
 
 	const winnerCards = winners.map(item => {
+		const image = winnerImageMap.get(item.id);
 		return (
-			<Grid container key={item.title} spacing={2} alignItems='center'
-				style={{ paddingTop: 60 }}>
-				<Grid item md={5} xs={12}>
-					<Img fluid={winnerImageMap.get(item.id)} />
-				</Grid>
-				<Grid item md={7} xs={12}>
-					<Typography variant='h5'>{item.title} –{' '}
-						<span style={{ fontStyle: 'italic' }}>
-							{item.category}
-						</span>
+			<Grid item key={item.title} sm={6} style={{ paddingBottom: isSmall ? theme.spacing(4) : theme.spacing(8) }}>
+				<Box display='flex' flexDirection='column'>
+					<Img fluid={{ ...image, aspectRatio: 1.5 }} style={{ borderRadius: '14px', marginBottom: '1em' }}/>
+					<Typography variant='h5'>{item.title}</Typography>
```

Comment:
Please check the heading level. This should probably be h2, while the subtitle below should be h3. Lastly, the h2 and h3 should be grouped together into a single hgroup element.